systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -975,25 +1117,94 @@ void testDynamicallySetKafkaLoggingLevels() {
             });
 
         LOGGER.info("Setting external logging INFO");
+
+        final String externalLogging;
+        if (TestKafkaVersion.compareDottedVersions(Environment.ST_KAFKA_VERSION, "4.0.0") < 0) {
+            // Kafka 3.9
+            externalLogging = "log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender\n" +
+                    "log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout\n" +
+                    "log4j.appender.CONSOLE.layout.ConversionPattern=%d{ISO8601} %p %m (%c) [%t]%n\n" +
+                    "log4j.rootLogger=INFO, CONSOLE\n" +
+                    "log4j.logger.kafka=INFO\n" +
+                    "log4j.logger.org.apache.kafka=INFO\n" +
+                    "log4j.logger.kafka.request.logger=WARN\n" +
+                    "log4j.logger.kafka.network.Processor=ERROR\n" +
+                    "log4j.logger.kafka.server.KafkaApis=ERROR\n" +
+                    "log4j.logger.kafka.network.RequestChannel$=WARN\n" +
+                    "log4j.logger.kafka.controller=TRACE\n" +
+                    "log4j.logger.kafka.log.LogCleaner=INFO\n" +
+                    "log4j.logger.state.change.logger=TRACE\n" +
+                    "log4j.logger.kafka.authorizer.logger=INFO";
+        } else {
+            externalLogging = """
```

Comment:
As I can see this on multiple parts we should definitely create some wrapper around this to avoid redundancy. At least create issue would be fine with me I think.

Reply:
Problem is that the content seems to be often different. So not sure how much of wrapping you can do there. If you don't mind, I would leave that for you guys once this is merged :-)